src/commands/push.js

```diff
@@ -29,8 +30,19 @@ class PushCommand extends ApifyCommand {
         let actorId;
         let actor;
         // User can override actor version and build tag, attributes in localConfig will remain same.
-        const version = flags.version || flags.versionNumber || localConfig.version || '0.0';
-        const buildTag = flags.buildTag || localConfig.buildTag;
+        const version = flags.version || flags.versionNumber || localConfig.version || DEFAULT_ACTOR_VERSION_NUMBER;
+        let buildTag = flags.buildTag || localConfig.buildTag;
+        // We can't add the latest tag to everything. If a user creates a new
+        // version, e.g. for testing, but forgets to add the tag, it would default
+        // to latest and his production runs might switch to test version ❌
+        // TODO: revisit this when we have better build tagging system on platform.
+        if (!buildTag && version === DEFAULT_ACTOR_VERSION_NUMBER) {
+            // It would be better to tag this `version-0.0` or similar,
+            // or even keep it tag-less, but the platform complains when
+            // actor does not have a build with a `latest` tag, so until
+            // that changes, we have to add it.
+            buildTag = 'latest';
```

Comment:
Would be great to also use some const for this eg DEFAULT_BUILD_TAG

Reply:
+1

Reply:
Thanks, added 👍, but I'm interested to hear if you think it's more readable now. To me splitting the comments and defining the constant away from its usage did not make the code clearer. I used to put constants everywhere, but I realized that 

E.g. I prefer:

```js
class Car(opts){
  const {
    topSpeedKmh = 200
  }
}
```

over

```js
// at the top of the file
const DEFAULT_TOP_SPEED = 200;

// lots of other code here

class Car(opts){
  const {
    topSpeedKmh = DEFAULT_TOP_SPEED
  }
}
```

or even

```js
await enqueueLinks({
  strategy: 'same-domain' // instead of ENQUEUEING_STRATEGIES.SAME_DOMAIN.
});
```

Although its true that in this case, I rely on TypeScript automatically checking the string literal for typos.

Do you have a rule / best practice for it on platform?